d2/src/main/java/com/linkedin/d2/balancer/D2ClientConfig.java

```diff
@@ -171,6 +171,8 @@ public class D2ClientConfig
   public XdsServerMetricsProvider _xdsServerMetricsProvider = new NoOpXdsServerMetricsProvider();
   public boolean loadBalanceStreamException = false;
 
+  public boolean xdsInitialResourceVersionEnabled = false;
```

Comment:
Huge nit but the field name is `initialResourceVersions`, not `initialResourceVersion`.
```suggestion
  public boolean xdsInitialResourceVersionsEnabled = false;
```